test/specs/route-createRules.js

```diff
@@ -0,0 +1,41 @@
+
+describe('PushStateTree createRule', function() {
+  'use strict';
+
+  describe('when rule /^servers(\/)?(.*)/', function() {
+    var pst;
+    var rule;
+    beforeEach(function() {
+      pst = new PushStateTree();
+      rule = pst.createRule({
+        id: 'servers',
+        rule: /^servers(\/)?(.*)/
+      });
+    });
+    it('should create the html node rule', function() {
+      expect(rule.outerHTML).toEqual('<pushstatetree-rule id="servers" rule="/^servers(\\/)?(.*)/"></pushstatetree-rule>');
```

Comment:
you can check if is a `instanceof HTMLElement`, them check if it contains the attribute `rule` with the `string` to be compared with.